### ISSUE TITLE
Unit table is hard to understand in mobile view

### ISSUE THREAD

**Issue by anders-schneider** [CityOfDetroit/bloom#461]
In mobile view, when the unit table (in the individual-listing page) has more than one row (e.g. some number of 1BR and some number of 2BR), the display stacks the rows in a way that I find somewhat confusing:

![image](https://user-images.githubusercontent.com/8754454/130692526-8de30b5e-d16a-4025-93c6-a1f62482af17.png)

(The space between the "Total Count" of 1BR and "Unit Type" for the 2BR is the only indication that we're in a new section.)

Is this what we want? Is there a better way to visually separate the rows in mobile view?

**Comment by anders-schneider** [comment]
@tomgebauer thoughts?

**Comment by tomgebauer** [comment]
In the responsive view, the table headers are changed to standard rows which removes the visual hierarchy that is seen on desktop from the view. I'd propose keeping the header styling from the desktop view, see attached
![image](https://user-images.githubusercontent.com/3886741/130693042-98329b63-de5f-4f54-a8a9-9e2ecf456fbf.png)

**Comment by anders-schneider** [comment]
Thanks Tom! That screenshot looks a lot better than what we currently have.

And your comment made me want to check what we currently have for the "all listings" view. It seems to have the same issue:

![image](https://user-images.githubusercontent.com/8754454/130695054-768c8c87-48ab-4f98-b730-ebfa1201f9ae.png)

Implementation note: I think both the "all listings" and "individual listing" views use the same React component (`GroupedTable`) so I think we can fix both by fixing that component.

**Comment by tomgebauer** [comment]
@anders-schneider you're correct–if we could make this change on both "all listings" and "individual listing" views, that would be optimal.

**Comment by ludtkemorgan** [comment]
This problem still exists
